tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
I don't think you need a GPU version, just bazel test on the entire directory should work (something like bazel test tensorflow/contrib/...)